crates/top/rerun/src/commands/rrd/filter.rs

```diff
@@ -111,32 +111,40 @@ impl FilterCommand {
                 Ok(msg) => {
                     let msg = match msg {
                         re_log_types::LogMsg::ArrowMsg(store_id, mut msg) => {
-                            if !should_keep_entity_path(&dropped_entity_paths, &msg.batch.schema())
+                            match re_sorbet::ChunkSchema::try_from(msg.batch.schema_ref().as_ref())
```

Comment:
Why is `re_sorbet` being mentioned here? Isn't `re_sorbet` all about Sorbet-formatted chunks, as opposed to this tool which manipulates streams of Rerun-formatted chunks?